bolt/lib/Passes/BinaryPasses.cpp

```diff
@@ -1441,6 +1458,22 @@ Error PrintProgramStats::runOnFunctions(BinaryContext &BC) {
       StaleSampleCount += SampleCount;
       ++NumAllStaleFunctions;
     }
+
+    if (opts::ShowDensity) {
+      uint64_t Size = Function.getSize();
+      // In case of BOLT split functions registered in BAT, executed traces are
+      // automatically attributed to the main fragment. Add up function sizes
+      // for all fragments.
+      if (IsHotParentOfBOLTSplitFunction)
+        for (const BinaryFunction *Fragment : Function.getFragments())
+          Size += Fragment->getSize();
+      double Density = (double)1.0 * Function.getExecutedBytes() / Size;
+      FuncDensityList.emplace_back(Density, SampleCount);
```

Comment:
If there is no special reason to use `SampleCount` here, how about using `ExecutedBytes` to be consistent?  We use the same value for FDO(https://github.com/llvm/llvm-project/blob/main/llvm/tools/llvm-profgen/ProfileGenerator.cpp#L807-L809).

Reply:
I'm neutral about using ExecutedBytes but we use SampleCount for other profile-wide stats, e.g. stale samples percent.

Reply:
> I'm neutral about using ExecutedBytes but we use SampleCount for other profile-wide stats, e.g. stale samples percent.

Sounds good. No strong option, it shouldn't affect to the density value.

Reply:
Don't have a good idea, but `ExecutedBytes` confused me as byte implied static size. It gives the impression of byte-wise coverage.  `TotalSamplesInBytes`/`SampleCountInBytes`?

Reply:
Comments above the variable and the method say it's dynamically executed bytes, but will rename them to avoid confusion.

> TotalSamplesInBytes/SampleCountInBytes?

What do you mean by that?

Reply:
> What do you mean by that? 

@aaupov I meant to give you suggestions on alternative names. 

> Comments above the variable and the method say it's dynamically executed bytes 

Good naming should not be confusing, without need to look at comments. `ExecutedBytes` is confusing as it really lead people to think it's coverage in bytes.